CMakeLists.txt

```
@@ -282,8 +282,8 @@ else()
     add_subdirectory(src/ponyc)
 
     add_subdirectory(test/libponyc)
+    add_subdirectory(test/runner)
     add_subdirectory(test/libponyc-run)
```

Maybe this naming would make sense?

```suggestion
    add_subdirectory(test/full-program-runner)
    add_subdirectory(test/full-programs)
```